ryan-meinzer.txt

```diff
@@ -0,0 +1,91 @@
+https://codepen.io/ryanmeinzer/pen/rNarWbb
+
+    1. What is Semantic HTML? 
+    semantic = meaning
+    2. What is HTML used for? 
+    structuring language for the web and apps to grow
+    3. What is an attribute and where do we put it? 
+    it's a property that can change of a component, putting in html for example as a characteristic of a page element
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    h1 is the main hearder tag - can use on a page for the main heading but may not be practical to use it more than once
```

Comment:
Good point about not using it not then once on a page

Reply:
thanks ; )